### PR TITLE
UI Setup for OSX Catalina v10.15.7/Ubuntu 20.04.2 LTS

### PR DESCRIPTION
**Pull request by AmmarYousefM**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,16 +8,14 @@ It *stores all keys locally* in the browser, *never exposing your keys to anyone
...
 
-On Ubuntu and OSX, the easiest way to install Node is to use the [Node Version Manager](https://github.com/creationix/nvm).
-
-To install NVM for Linux/OSX, simply copy paste the following in a terminal:
+To install NODE for Linux/Ubuntu 20.04.2 LTS, simply copy paste the following in a terminal:
 
 ```
-curl -o- https://raw.githubusercontent.com/creationix/nvm/v0.30.2/install.sh | bash
-nvm install v9
-nvm use v9
```
Comment (sschiessl-bcp) [inline README.md:20]: Why did you remove the usage of nvm?

File: README.md [hunk 2]
```diff
@@ -30,7 +28,8 @@ cd bitshares-ui
 Before launching the GUI you will need to install the npm packages:
 
 ```
-npm install
+npm ci
```
Comment (sschiessl-bcp) [inline README.md:31]: ci is recommended to use for production builds. For development do `npm install`.

In both cases `install-charting-library` is called from `postinstall` script.

File: README.md (hunk 1 above)
Comment (AmmarYousefM) [reply README.md:20]: Perhaps it's not needed in the UI user setup procedure; switching between node versions or "Node Version Manager" is perhaps an extra admin server feature and not a requirement for setup the UI.

File: README.md (hunk 2 above)
Comment (AmmarYousefM) [reply README.md:31]: This was tested over *Ubuntu 20.04.2 LTS (GNU/Linux 5.4.0-73-generic x86_64)*:

*In both cases *install-charting-library* isn't getting installed during postinstall script with a warning says:*

npm WARN lifecycle BitShares2-light@5.0.20210720-rc1~postinstall: cannot run in wd BitShares2-light@5.0.20210720-rc1 npm run install-charting-library (wd=/root/bitshares-ui)

The above will prevent the build from completing later with an error:

ERROR in ./app/components/Exchange/TradingViewPriceChart.jsx
Module not found: Error: Can't resolve '../../../charting_library/charting_library.esm' in '/root/bitshares-ui/app/components/Exchange'
 @ ./app/components/Exchange/TradingViewPriceChart.jsx 10:18-75
 @ ./app/components/Exchange/Exchange.jsx
 @ ./app/components/Exchange/ExchangeContainer.jsx
 @ ./app/App.jsx
 @ ./app/AppInit.jsx
 @ ./app/index.js
 @ ./app/Main.js

File: README.md [hunk 3]
```diff
@@ -8,19 +8,17 @@ It *stores all keys locally* in the browser, *never exposing your keys to anyone
 
 ## Getting started
 
-BitShares-UI depends node Node.js, and version 8+ is required.
+BitShares-UI depends node (Node.js), we recommend using node version 10.
```
Comment (abitmore) [inline README.md:11]: I think the correct grammar is "depends on".

File: README.md (hunk 1 above)
Comment (abitmore) [reply README.md:20]: @AmmarYousefM although I agree it's good to install nodejs via apt on Ubuntu, you removed instructions about macOS (formerly OSX), which is not very good (btw I didn't test whether the old instruction would still work on macOS).

File: README.md (hunk 2 above)
Comment (abitmore) [reply README.md:31]: Thanks for the details. I guess it's better to fix the `postinstall` script?

Comment (abitmore) [reply README.md:31]: According to this link: https://stackoverflow.com/questions/18136746/npm-install-failed-with-cannot-run-in-wd , I guess you got the error because you were installing with `root`?

File: README.md (hunk 1 above)
Comment (AmmarYousefM) [reply README.md:20]: @abitmore will be testing the same commands over my OSX and get back to this PR with an update.

File: README.md (hunk 2 above)
Comment (AmmarYousefM) [reply README.md:31]: @abitmore thanks, yes was installing using root user on a fresh installed server; I agree it's not a good practice to dvelop under root, perhaps writing a note, or changing the procedure is helpful in this case as users are facing the issue and last time was reported yesterday on telegram, perhaps generalizing a single commend such as *npm install --unsafe-perm* is a solution but maybe "unsafe" looks ugly, more suggestions are appreciated.

Comment (abitmore) [reply README.md:31]: Yes it does look ugly. :P I think it's OK to add a note about root though. @sschiessl-bcp any idea?

Comment (sschiessl-bcp) [reply README.md:31]: I'd say best would be to add a note to do this with a non-root user

File: README.md (hunk 1 above)
Comment (AmmarYousefM) [reply README.md:20]: I had tested the below procedure over OSX/macOS Catalina v10.15.7 (Latest) and it went successfully:

curl -o- https://raw.githubusercontent.com/creationix/nvm/v0.30.2/install.sh | bash
nvm install v10
nvm use v10
git clone https://github.com/bitshares/bitshares-ui.git
cd bitshares-ui
npm install
npm run build

Note: since nvm can be generalized across OSX and Ubuntu, it's good to keep it in place rather than writing two procedures for every OS.
